src/homes/queryBuilder.ts

```diff
@@ -5,8 +5,6 @@ import { ISearchParams } from "../types"
 
 const queryBuilder = (params: ISearchParams) => {
   return communitiesQueryBuilder(params)
-    .replace(/model_type_ids/g, "model_type_id%5B%5D")
```

Comment:
Next time we do updates, let's consider removing this file and moving the communities queryBuilder into its parent directory.